### PR TITLE
Creating story for `asset-list`

### PR DESCRIPTION
**Pull request by dhruvv173**
## Explanation

- Created story for `asset-list` component
- `ui/components/app/aasset-list`

### Before

Screenshot 2023-06-05 225337

### After

image
image

https://github.com/MetaMask/metamask-extension/assets/79097544/daca693e-395a-497c-998a-ea843bc441de

covered `jest` test
image

## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- Search for `asset-list` component

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/asset-list/asset-list.stories.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import AssetList from '.';
+
+export default {
+  title: 'Components/App/AssetList',
+
```
Comment (georgewrmarshall) [inline ui/components/app/asset-list/asset-list.stories.js:6]: nit: remove line break
```suggestion
```

**Comment by dhruvv173** [comment]
Hey @georgewrmarshall, I was going through the codebase and noticed that there are several components for which stories have not been created. I have some experience in creating new stories, as I have previously created some during my PRs with your assistance. I have created one for the `asset-list` component. I hope it is acceptable that I took the initiative. I apologize in advance if it was not appropriate. Could you please review this PR? 
Thanks!

**Review by georgewrmarshall — COMMENTED** [review]
Hey @dhruvv173, thanks for taking the initiative. Yes, I absolutely think all components should have stories. Looks like you have one small linting issue `yarn lint` otherwise LGTM! Thanks for your contribution

**Comment by dhruvv173** [comment]
> Hey @dhruvv173, thanks for taking the initiative. Yes, I absolutely think all components should have stories. Looks like you have one small linting issue `yarn lint` otherwise LGTM! Thanks for your contribution

Can I go ahead and try to make stories for the missing components?

**Comment by georgewrmarshall** [comment]
Hey @dhruvv173, I think that would be ok. You could start with the `ui/components/ui` folder?

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Comment by dhruvv173** [comment]
> Hey @dhruvv173, I think that would be ok. You could start with the `ui/components/ui` folder?

Sure, will do!

**Review by garrettbear — APPROVED** [review]

**Pull request merged**